components/clarinet-cli/src/frontend/cli.rs

```diff
@@ -422,6 +426,9 @@ struct Integrate {
         conflicts_with = "use-on-disk-deployment-plan"
     )]
     pub use_computed_deployment_plan: bool,
+    /// Path to Package.json
+    #[clap(long = "package")]
+    pub package: Option<String>,
```

Comment:
I think I'm missing some context here.
What is the package.json used for?

Reply:
last bullet from #1101 

> This new --package flag will allow a user to start a devnet using only a clarinet project package (created from clarinet devnet package). The user can then start a devnet without having to run clarinet integrate from a specific directory - they just need the package file, and it has everything they need.

Reply:
ah yes ofc. My bad, I was thinking about a node.js `package.json` and got confused. Sounds good then, thx